com.github.micahflee.torbrowser-launcher.yaml

```diff
@@ -1,6 +1,6 @@
 app-id: com.github.micahflee.torbrowser-launcher
 runtime: org.kde.Platform
-runtime-version: 5.15-21.08
+runtime-version: 5.15-22.08
 sdk: org.kde.Sdk
 base: com.riverbankcomputing.PyQt.BaseApp
```

Comment:
This PR needs to wait until the `com.riverbankcomputing.PyQt.BaseApp` gets updated to 5.15-22.08.

Reply:
It is now updated

Reply:
Thanks!